Source/Core/DolphinQt2/MainWindow.h

```diff
@@ -145,6 +148,8 @@ class MainWindow final : public QMainWindow
   NetPlayDialog* m_netplay_dialog;
   NetPlaySetupDialog* m_netplay_setup_dialog;
   GraphicsWindow* m_graphics_window;
+  std::array<GCTASInputWindow*, 4> m_gc_tas_input_windows{};
```

Comment:
Maybe replace the magic number with a ``constexpr`` constant?

Reply:
I dont feel thats necessary as the number isnt very magic, the gc has 4 controllers so there are 4 windows.

Reply:
Sure you can understand it but you have to make the connection first. At first I wasn't even sure why there should be 4 windows. Not something worth blocking over but I do think it makes the code easier to understand.